.github/scripts/trymerge.py

```diff
@@ -1265,6 +1293,85 @@ def checks_to_str(checks: List[Tuple[str, Optional[str]]]) -> str:
 def checks_to_markdown_bullets(checks: List[Tuple[str, Optional[str]]]) -> List[str]:
     return [f"- [{c[0]}]({c[1]})" if c[1] is not None else f"- {c[0]}" for c in checks[:5]]
 
+def get_flaky_rules() -> List[FlakyRule]:
+    url = "https://raw.githubusercontent.com/pytorch/test-infra/generated-stats/stats/flaky-rules.json"
```

Comment:
This makes this super specific to a pytorch, can we perhaps make it a merge rules config?
Say `flake_rules_location_url` property?